spec/requests/portfolios_rbac_write_access_spec.rb

```diff
@@ -9,15 +9,19 @@
 
   describe "POST /portfolios" do
```

Comment:
Should the name of this spec file be changed? With pulling out the `write` permission possibly `portfolios_rbac_modify_access_spec ( something without the word 'write' in it. )